packages/native-stack/src/views/NativeStackView.native.tsx

```diff
@@ -513,95 +531,122 @@ export function NativeStackView({ state, navigation, descriptors }: Props) {
 
   const { colors } = useTheme();
 
+  const currentRouteKey = state.routes[state.index].key;
+  const topScreenOptions = descriptors[currentRouteKey].options;
+  const gestureDetectorBridge = React.useRef<GestureDetectorBridge>({
+    stackUseEffectCallback: (_stackRef) => {
+      // this method will be overriden in GestureDetector
+    },
+  });
```

Comment:
is it necessary to specify object with empty method here instead of `null`?

```suggestion
  const gestureDetectorBridge = React.useRef<GestureDetectorBridge>({
    stackUseEffectCallback: (_stackRef) => {
      // this method will be overriden in GestureDetector
    },
  });
```

Reply:
It must stay with defined object, since we're referring it directly in ScreenGestureDetector/ScreenStack: https://github.com/software-mansion/react-native-screens/blob/546e72f56359eacade1eb35700dcb19aa8434f66/src/gesture-handler/ScreenGestureDetector.tsx#L63.

Reply:
@tboba I see, it's a bit strange.

That code also looks problematic. Refs shouldn't be assigned during render https://react.dev/learn/referencing-values-with-refs#best-practices-for-refs

Though that's out of scope of this PR.